packages/mosaic/select/select.component.ts

```diff
@@ -902,11 +902,15 @@ export class McSelect extends McSelectMixinBase implements
 
         let totalItemsWidth: number = 0;
         const itemMargin: number = 4;
-        triggerClone.querySelectorAll('mc-tag').forEach((item) => {
-            totalItemsWidth += item.getBoundingClientRect().width as number + itemMargin;
-        });
 
-        triggerClone.remove();
+        const mcTags = triggerClone.querySelectorAll('mc-tag');
+        const mcTagsLength = mcTags.length;
+
+        for (let i = 0; i < mcTagsLength; i++) {
```

Comment:
why did you change 'forEach' to 'for' ?

Reply:
@lskramarov forEach call is not supported for querySelectorAll result in IE11 (updated the description of PR).

Reply:
for IE11 support using Polyfills like "core-js"

for example,
https://github.com/Teradata/covalent/blob/master/docs/IE11.md

there is an example - IE11 Flexbox Workarounds

So, you can leave methods - remove() and forEach()

Reply:
Guys, @lskramarov @Fost 
here is what I found out.

1) core-js author added  iterable DOM collection support (`forEach()`) to the 3th version of core-js. [PR for this](https://github.com/zloirock/core-js/pull/325) 

 Here we have 3 solutions: 
- update core-js (what are drawbacks for this?) to the current version 3.x.x;
- add polyfill by ourselves;
- rewrite to `Array.prototype.forEach`;
- keep the current solution;

2) It seems that `element.remove()` method is not the part of core-js (i've looked through the source code) neither babel (which is used by core-js, as I see): https://github.com/babel/babel/issues/8406
Here are solution:
- rewrite to `parent.removeChild(...)` native syntax
- keep the current solution
(optional) open an issue on adding this functionality into core-js.


About polyfills. @Fost, it seems we don't need make additional imports, since they are provided with webpack (see a screenshot). Correct me if I'm wrong.
![image](https://user-images.githubusercontent.com/1513274/63694362-f070a900-c81e-11e9-8688-deb0fe3a8f0d.png)

Reply:
**But**,
webpack config using only for local development, another words its syntetic and sometimes unstable environment. Build process not using core-js or webpack or smth else.

Mosaic packages have source in format ESM & FESM.
`core-js` using only in web-application if they need it support IE11 (or other oldest browsers).